cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
```

Comment:
Do not push file with only comments

Reply:
![image](https://user-images.githubusercontent.com/43115836/121594341-538ef800-ca13-11eb-9876-401684eee086.png)
Those comments help a lot, they explain how commands.js works on cypress

Reply:
I will need them later